rules-tests/TypeDeclaration/Rector/ClassMethod/ArrayShapeFromConstantArrayReturnRector/Fixture/skip_key_with_colon_multi.php.inc

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\ArrayShapeFromConstantArrayReturnRector\Fixture;
+
+final class SkipKeyWithColonMulti
+{
+    public function run(string $name)
+    {
+        return ['prefix:key' => $name, 'another:key' => $name];
```

Comment:
I've noticed also these fails:

```php
return ['{key}' => $name];
```

Reply:
I created PR https://github.com/rectorphp/rector-src/pull/1968 for it.